configure.ac

```diff
@@ -238,6 +238,31 @@ AC_TRY_LINK([#include <stdint.h>],
     ],
     [AC_MSG_RESULT(no)])
 
+dnl Check for gcc memory model aware built-in atomics
+dnl If supported check to see if not internal to compiler
+AC_MSG_CHECKING(compiler support for built-in memory model aware atomics)
+AC_TRY_LINK([#include <stdint.h>],
```

Comment:
FWIW: `AC_LINK_IFELSE` is preferred to `AC_TRY_LINK` these days.

Reply:
@jsquyres - Thank you, that does appear to be the case. Since AC_TRY_LINK is used throughout the existing file I'm hesitant to introduce a change to move to AC_LINK_IFELSE within this PR. Possibly a more generic cleanup of the configure.ac file should done in a separate PR post 1.7rc2?